ui/components/ui/site-icon/site-icon.js

```diff
@@ -3,16 +3,20 @@ import PropTypes from 'prop-types';
 import IconBorder from '../icon-border';
 import IconWithFallback from '../icon-with-fallback';
 
-export default function SiteIcon({ icon = null, name = '', size }) {
+export default function SiteIcon({ icon = null, name = '', size, className }) {
   const iconSize = Math.floor(size * 0.75);
   return (
-    <IconBorder size={size}>
+    <IconBorder size={size} className={className}>
       <IconWithFallback icon={icon} name={name} size={iconSize} />
     </IconBorder>
   );
 }
 
 SiteIcon.propTypes = {
+  /**
+   * Additional className to add to the root element of SiteIcon
+  */
+  className: PropTypes.string,
```

Comment:
```suggestion
  /**
   * Additional className to add to the root element of SiteIcon
   */
```

For alignment lint failure

Reply:
Thanks for pushing a fix. The suggested change removed the `SiteIcon.propTypes = {`and caused the build to fail. I added it back to your commit and it should now pass the linting step.